languages/csharp/config.json

```diff
@@ -166,6 +172,18 @@
         "concepts": ["randomness"],
         "prerequisites": ["numbers"]
       },
+      {
+        "slug": "regular-expressions",
+        "uuid": "c63bb75d-51bd-4d5d-b37d-88e53ee0da69",
+        "concepts": ["regular-expressions"],
+        "prerequisites": [
+          "arrays",
+          "for-loops",
+          "strings",
+          "string-interpolation",
+          "verbatim-strings"
```

Comment:
We currently don't have separate `string-interpolation` and `verbatim-strings` concepts being unlocked, but we could add these to the `strings` exercise.

Reply:
Both are concepts in `string-formatting`.

Reply:
Brilliant. Thanks